### ISSUE TITLE
Exact

### ISSUE THREAD

**Issue by AgarioCheats** [Agarian/How-do-i-make-Ogar-agar.io-custom-gamemode#1]
We're looking for the same feature! Smooth split like in Agar.io ->  Been searching all over. Seems like it's a top secret code.

**Comment by Agarian** [comment]
intrested to know how its done

**Comment by AgarioCheats** [comment]
Yes i've seen it. I've worked multiple days trying to get it to work, so far.

**Comment by Agarian** [comment]
theres a line of code with *splitSpeed* in *Gameserver.js*. do Ctrl+F to find the line. Theres two sets of them.

**Comment by AgarioCheats** [comment]
Yep, i've been looking at them for hours, you don't know. But the thing is, changing any values there won't help. I know what every value does, but they won't do anything because a new code must be entered in.

**Comment by Agarian** [comment]
yeah i know i thought u knew a way here
